frontend/components/Button/Button.tsx

```diff
@@ -0,0 +1,57 @@
+import Link from "next/link";
+
+export type ButtonProps = {
+    /** The text inside the Button */
+    children: string
+    /** Which type of button we want, tertiary is a link */
+    kind?: "primary" | "secondary" | "tertiary"
+    /** Button type default is "button" */
+    type?: "button" | "reset" | "submit"
```

Comment:
Ist es hier möglich diese Type splittung zu verwenden? Das wir sagen type und onClick sind zusammen oder to? Verstehst du was ich meine. Glaube man codet das irgendwie so:

```ts
type ButtonBase = ....
type ButtonOnClick = ...
type ButtonTo = ...
type ButtonProps = (ButtonTo | ButtonOnClick) & ButtonBase
```